packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -154,10 +156,26 @@ export class ProxyContext implements EditableTreeContext {
         return this.getRoot(true);
     }
 
+    public set unwrappedRoot(value: ContextuallyTypedNodeData | undefined) {
+        const rootField = this.getRoot(false);
+        const mapTrees = applyFieldTypesFromContext(this.schema, rootField.fieldSchema, value);
+        const cursors = mapTrees.map(singleMapTreeCursor);
+        if (rootField.length > 0) {
+            rootField.replaceNodes(0, cursors);
+        } else {
+            rootField.insertNodes(0, cursors);
+        }
+    }
+
     public get root(): EditableField {
         return this.getRoot(false);
     }
 
+    public set root(value: ContextuallyTypedNodeData | undefined) {
+        assert(isArrayLike(value), "expected array data");
```

Comment:
Why do we always expect an array when setting the root?

Reply:
> Why do we always expect an array when setting the root?

Good point. I have removed this and also updated the documentation to reflect the new API.